shared/src/main/scala/mlscript/TyperHelpers.scala

```diff
@@ -1077,8 +1077,8 @@ abstract class TyperHelpers { Typer: Typer =>
         // * Object types do not need to be completed in order to be expanded
         info.kind.isInstanceOf[ObjDefKind]
         || info.isComputed)
-    // TODO add pol: Bool
-    def expand(implicit ctx: Ctx, raise: Raise): SimpleType = {
+    // TODO provide correct pol at each expand call (we're assuming `pol = true` now)
```

Comment:
> (we're assuming `pol = true` now)

Just saw that. Really not a good idea as you'll forget to fix most of the places you put the wrong value...

Reply:
now all fixed...

Reply:
now all fixed...